test/sbvr/pilots.js

```diff
@@ -26,6 +26,8 @@ describe('pilots', function () {
 		`\
 Term:      name
 	Concept Type: Short Text (Type)
+Term:      nickname
+	Concept Type: Short Text (Type)
```

Comment:
I needed a nullable term to actually be able to test we generated sensible rules as opposed to having them all optimized away to nothing because they couldn't be broken :|